### PR TITLE
[Build scripts] United mediasdk_directories

### PR DESCRIPTION
**Pull request by dlobanox**

### CODE REVIEW

File: common/mediasdk_directories.py [hunk 1]
```diff
@@ -52,60 +115,90 @@ def find_folder_on_disks(folder):
...
-        # linux_open_source
-        'MediaSDK': 'https://github.com/Intel-Media-SDK/MediaSDK',
-        'libva': 'https://github.com/intel/libva',
-        # test_repositories
-        'flow_test': 'https://github.com/Intel-Media-SDK/flow_test',
-    }
+    _next_gen_tests_root_url = urljoin(_closed_root_url, 'next_gen_tests')
+    _next_gen_builds_root_url = urljoin(_closed_root_url, 'next_gen_builds')
+
+    if platform.system() == OsType.windows:
```
Comment (azhogov) [inline common/mediasdk_directories.py]: why do you need such if (and below too)?

File: common/mediasdk_directories.py [hunk 2]
```diff
@@ -52,60 +115,90 @@ def find_folder_on_disks(folder):
...
     """
-    _mount_point_root_dir = pathlib.Path(r'/media')
-    _tests_root_path = _mount_point_root_dir / 'tests'
-    _builds_root_path = _mount_point_root_dir / 'builds'
+    """Base class for all MediaSDK dirs errors"""
+
+    _share_paths = SHARE_PATHS
+    _repositories = REPOSITORIES
 
+    _closed_root_url = r'http://bb.msdk.intel.com'
```
Comment (azhogov) [inline common/mediasdk_directories.py]: it should be in method get_root_url()

File: common/mediasdk_directories.py [hunk 3]
```diff
@@ -52,60 +115,90 @@ def find_folder_on_disks(folder):
...
+
     """
     Container for static links
     """
-    _mount_point_root_dir = pathlib.Path(r'/media')
-    _tests_root_path = _mount_point_root_dir / 'tests'
-    _builds_root_path = _mount_point_root_dir / 'builds'
+    """Base class for all MediaSDK dirs errors"""
+
+    _share_paths = SHARE_PATHS
```
Comment (azhogov) [inline common/mediasdk_directories.py]: are not needed here

File: common/mediasdk_directories.py [hunk 4]
```diff
@@ -52,60 +115,90 @@ def find_folder_on_disks(folder):
...
+    _repositories = REPOSITORIES
 
+    _closed_root_url = r'http://bb.msdk.intel.com'
     _public_root_url = r'http://mediasdk.intel.com'
-    _private_root_url = r'http://bb.msdk.intel.com'
 
     _public_tests_root_url = urljoin(_public_root_url, 'tests')
```
Comment (azhogov) [inline common/mediasdk_directories.py]: all *_root_url should be in corresponding methods

File: ted_adapter/test_adapter.py [hunk 5]
```diff
@@ -189,8 +189,18 @@ def main():
...
-    parser.add_argument('-p', "--product-type", default='linux',
-                        choices=['linux', 'embedded', 'open_source', 'windows', 'api_latest'],
+    parser.add_argument('-p', "--product-type", default='closed_linux',
+                        choices=[
+                            # closed
+                            'closed_windows', 'closed_windows_hw_lib', 'closed_windows_tools',
+                            'closed_windows_sw_lib', 'closed_windows_mfts', 'closed_windows_uwp',
+                            'closed_linux', 'closed_linux_open_source', 'closed_embedded', 'closed_android',
+                            # private
+                            'private_android', 'private_linux_next_gen', 'private_linux_next_gen_api_latest',
```
Comment (azhogov) [inline ted_adapter/test_adapter.py]: *api_next

File: common/static_public_data.py [hunk 6]
```diff
@@ -0,0 +1,10 @@
+SHARE_PATHS = {'build_linux': r'/media/builds',
+               'test_linux': r'/media/tests', }
+
+REPOSITORIES = {
+    # linux_open_source
+    'MediaSDK': 'https://github.com/Intel-Media-SDK/MediaSDK',
+    'libva': 'https://github.com/intel/libva',
+    # test_repositories
+    'flow_test': 'git@github.com:Intel-Media-SDK/flow_test.git',
```
Comment (GoAlexander) [inline common/static_public_data.py]: ssh? Are you sure?

**Pull request merged**